src/TVar.cpp

```diff
@@ -211,6 +211,11 @@ bool TVar::setName(const QString& n)
     return true;
 }
 
+void TVar::setId(const int Id)
```

Comment:
See my comment about the header file - you don't need this method as something similar already exists for this class inherited from the template in the base `Tree` class that it is derived from.